<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cliente</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPuntaje">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cronometro">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>420</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>471</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="tarjeta17">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>170</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>3x5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta23">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>250</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>4x5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta03">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>1x3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta20">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>4x2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta29">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>330</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>5x5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta12">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>2x6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta18">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>170</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>3x6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta24">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>250</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>4x6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta01">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>1x1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta25">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>5x1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta07">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>2x1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta30">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>330</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>5x6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta10">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>2x4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta26">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>330</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>5x2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta09">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>2x3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta28">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>330</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>5x4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta15">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>3x3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>3x1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta16">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>3x4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta05">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>1x5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta22">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>250</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>4x4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta06">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>1x6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta19">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>4x1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta08">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>2x2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta11">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>2x5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta14">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>3x2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta02">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>1x2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta21">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>4x3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta27">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>330</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>5x3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tarjeta04">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>1x4</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
